src/main/kotlin/fi/metatavu/oss/api/impl/pages/PageEntity.kt

```diff
@@ -24,6 +25,12 @@ class PageEntity : DBMetadata() {
     @ManyToOne(optional = false)
     lateinit var survey: SurveyEntity
 
+    @ManyToOne(optional = true)
+    var layout: LayoutEntity? = null
+
+    @Column
+    var orderNumber: Int? = null
```

Comment:
should the order be required? It would make it easier to ensure that the order remains stable